packages/angular_devkit/build_angular/src/server/index.ts

```diff
@@ -59,6 +59,14 @@ export function execute(
   const baseOutputPath = path.resolve(root, options.outputPath);
   let outputPaths: undefined | Map<string, string>;
 
+  if (options.bundleDependencies === 'none' && tsConfig.options.enableIvy) {
+    context.logger.warn(tags.stripIndent`
```

Comment:
I think this falls within the category of "warnings you can't turn off are bad and people ignore them". But I also don't think it warrants another option in the config.

Can we maybe actually check instead? We know when we're using Ivy. And we can know if Ivy processed a package by checking something known, like `@angular/core`. Maybe the right thing to do is check if these match? But then we break the local lib development journey because it compiles directly to Ivy.

This does feel like an extremely rare case so maybe just always warning is ok.

Reply:
I think we can do it, as it's trivial.

Reply:
I hadn't really considered we could just do it here in that way. Yeah that LGTM, now the only way to see the warning is if you use a precompiled angular/core for server builds. Which is very unlikely.